Cinebase/Use Cases/Movies/MovieDetail/View/MovieDetailViewController.swift

```diff
@@ -51,7 +55,9 @@ class MovieDetailViewController: UIViewController, MovieDetailView {
     }
     
     @IBAction func handlePlayTrailer(_ sender: Any) {
-        presenter.playTrailer()
+        if let movieID = movieID {
+            presenter.playTrailerwithID(movieID)
```

Comment:
If the presenter does the transition, we wouldn't need the movieID on the movie view model